src/schema/schema_worker.c

```diff
@@ -95,10 +93,8 @@ __schema_layered_worker_verify(WT_SESSION_IMPL *session, const char *uri,
     WT_ASSERT(session, file_func == __wt_verify);
 
     /*
-     * Verifying stable tables of layered tables uses the existing verify logic.
-     * Ingest tables, however, require special handling:
-     * - On leader: ingest must always be empty/no-op.
-     * - On followers: FIXME-WT-15047 ingest tables are not checkpointed.
+     * Verifying stable tables of layered tables uses the existing verify logic. Ingest tables,
+     * however, must be empty of no-op. We only verify the ingest tables of leaders.
```

Comment:
What do you think of the following?
```suggestion
     * Verifying stable tables of layered tables uses the existing verify logic. The same applies to ingest tables of leaders. However, on followers ingest tables must be empty.
```